README.md

```diff
@@ -142,7 +142,7 @@ repositories {
 String findRustlsPlatformVerifierProject() {
     def dependencyText = providers.exec {
         it.workingDir = new File("../")
-        commandLine("cargo", "metadata", "--format-version", "1", "--manifest-path", "$PATH_TO_DEPENDENT_CRATE/Cargo.toml")
+        commandLine("cargo", "metadata", "--format-version", "1", "--filter-platform", "aarch64-linux-android", "--manifest-path", "$PATH_TO_DEPENDENT_CRATE/Cargo.toml")
```

Comment:
Should the text above be updated to mention the `--filter-platform`? I assume for larger projects it's probably something that can't be fixed to just `aarch64-linux-android` and needs some special consideration (?)

Reply:
I can't think of any reason you wouldn't be able to hardcode this. ARM64 is the largest platform by far in the Android device ecosystem, so every application is going to target at it at minimum (and likely ARMv7 and x86_64 too), making this a safe bet to pick up on the relevant `rustls-platform-verifier` dependency.